mephisto/abstractions/blueprints/remote_procedure/remote_procedure_agent_state.py

```diff
@@ -92,6 +97,11 @@ def load_data(self) -> None:
             self.final_submission = state["final_submission"]
             self.start_time = state["start_time"]
             self.end_time = state["end_time"]
+            # Metadata field would not exist prior to this pr
+            if "metadata" not in state:
+                self.metadata = {"tips": [], "feedback": []}
```

Comment:
Was running into some weird scenarios when switching branches and running tasks, in which I would get a key error with metadata when loading data. That's why I added this if block

Reply:
Yeah, I just removed the if statement and I don't see any key errors. I'll just get rid of the check.